source/_patterns/01-atoms/button/index.js

```diff
@@ -0,0 +1,12 @@
+// import $ from 'jquery';
+// might be good to force in some jquery here
+import button from 'bootstrap/js/src/button';
+
+require('./button.scss');
+
+export default {
+  button: [button],
```

Comment:
What is this key?

Reply:
This is to pull in button.js. If I don't include the brackets then the linter hits me with this error:
`8:3  error  Expected property shorthand  object-shorthand`